lib/_http_outgoing.js

```diff
@@ -140,6 +142,12 @@ function OutgoingMessage() {
 ObjectSetPrototypeOf(OutgoingMessage.prototype, Stream.prototype);
 ObjectSetPrototypeOf(OutgoingMessage, Stream);
 
+ObjectDefineProperty(OutgoingMessage.prototype, 'bytesWritten', {
+  get() {
+    return this[kBytesWritten];
+  }
+});
```

Comment:
Could this break userland which extend e.g. `ServerResponse` and add this themselves?

Reply:
Not sure. Maybe some web framework listed in https://github.com/nodejs/citgm/blob/main/lib/lookup.json? I think we could also try some HTTP clients like `got`.

Reply:
I added a setter as well to try and avoid breaking